src/layer/__tests__/cache.ts

```diff
@@ -496,3 +497,56 @@ describe('Unit test for cacheStillValid', () => {
     expect(isValid).toBeFalsy();
   });
 });
+
+describe('Unit test for aux request caching', () => {
+  beforeEach(async () => {
+    Object.assign(global, makeServiceWorkerEnv(), fetch); // adds these functions to the global object
+    await invalidateCaches();
+    setAuthToken(undefined);
+  });
+  it('It should be cache aux request to memory by default', async () => {
+    setAuthToken(EXAMPLE_TOKEN);
+    const { layer, mockedResponse, expectedLayerParams } = constructFixtureUpdateLayerFromServiceIfNeeded({});
+    const requestsConfig = {
```

Comment:
You could expand this test a bit by using  it.each and use different requestsConfigs as an input
listOfRequstConfigs=[
{} , 
null, 
undefined, 
{ retries: 1 },
{
      cache: {
        expiresIn: 60,
      },
   }
... all other combination of expiresIn and target params,
]
  
 it.each([....listOfRequstConfigs])('It should be cache aux request to memory by default', async requestsConfig => {....})

fftr

Reply:
I expanded the test as suggested and added other `requestsConfigs`